kvdb-rocksdb/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+### Changed
+- Default column support removed from the API
+  - Column argument type changed from `Option<u32>` to `u32`
+  - Migration
+    - Column index `None` -> `0`, `Some(0)` -> `1`, `Some(1)` -> `2`, etc.
+    - Database must be opened with at least one column
```

Comment:
```suggestion
    - Database must be opened with at least one column and existing DBs has to be opened with a number of columns increased by 1 to avoid having to migrate the data, e.g. before: `Some(9)`, after: `10`.
```